product-mini/platforms/windows/CMakeLists.txt

```
@@ -137,7 +137,7 @@ add_executable (iwasm main.c ${UNCOMMON_SHARED_SOURCE})
 
 install (TARGETS iwasm DESTINATION bin)
 
-target_link_libraries (iwasm vmlib ${LLVM_AVAILABLE_LIBS} ${UV_A_LIBS})
+target_link_libraries (iwasm vmlib  ${LLVM_AVAILABLE_LIBS} ${UV_A_LIBS} )
```

```suggestion
target_link_libraries (iwasm vmlib ${LLVM_AVAILABLE_LIBS} ${UV_A_LIBS} )
```
(formatting - extra space)